spec/core/block/routes/getMerkleTransaction.yml

```diff
@@ -3,9 +3,16 @@ tags:
 summary: Get the merkle path for a given a transaction and block
 description: |
   Returns the merkle path for a transaction included in a block.
-  The path is the complementary data needed to calculate the merkle root.
-  A client can compare if the calculated root equals the one recorded in the
-  block header, verifying that the transaction was included in the block.
+  The merkle path is the minimum number of nodes needed to calculate the merkle root.
+
+  Steps to calculate the merkle root:
+  1. proofHash = hash (leaf)
+  2. Contatentate and hash the first item from the merklePath list:
```

Comment:
concatenate*, i'd suggest the following so that steps are clear and not confused (verb hash with hash element)

Concatenate, and then hash, the first item from the merklePath list: